https://github.com/llvm/llvm-project/commit/f51ab1871655a9a96134c2636c37dcb5a6b01ac3
changed to use LLVM_EXTERNAL_CLANG_TOOLS_EXTRA_SOURCE_DIR
but this is not defined anywhere


--- a/tools/clang/tools/extra/clangd/quality/CompletionModel.cmake.orig	2021-04-01 09:20:32.000000000 -0700
+++ b/tools/clang/tools/extra/clangd/quality/CompletionModel.cmake	2021-04-01 09:20:47.000000000 -0700
@@ -5,7 +5,7 @@
 # will define a C++ class called ${cpp_class} - which may be a
 # namespace-qualified class name.
 function(gen_decision_forest model filename cpp_class)
-  set(model_compiler ${LLVM_EXTERNAL_CLANG_TOOLS_EXTRA_SOURCE_DIR}/clangd/quality/CompletionModelCodegen.py)
+  set(model_compiler ${CMAKE_SOURCE_DIR}/../clang-tools-extra/clangd/quality/CompletionModelCodegen.py)
 
   set(output_dir ${CMAKE_CURRENT_BINARY_DIR})
   set(header_file ${output_dir}/${filename}.h)
